loklak.py

```diff
@@ -84,7 +84,7 @@ def csvToJson(self, csvData = None, fieldnamesList = None):
         if csvData:
             data = csv.DictReader( csvData, fieldnames = fieldnamesList)
             jsonData = json.dumps( [ row for row in data ] )
-            return out
+        return jsonData
```

Comment:
if the code doesn't enter the `csvData` if block, it ends up returning `''` which is not a valid json. if the JSON is what needs to be returned then ideally it should return `{}`

Reply:
Then that should be applied to `xmlToJson` method too. 
This method not at all gives expected output json output. Should I change the whole definition of this method?(which can generate right json data from csv)

Reply:
Absolutely! Thanks for catching that. Feel free to make a change to that too. 😄